src/simplewallet/simplewallet.cpp

```diff
@@ -2415,8 +2429,39 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     payment_id_seen = true;
   }
 
+  std::string supernode_public_id;
+  cryptonote::account_public_address supernode_public_address;
+
+  if (transfer_type == TransferStake)
+  {
+    supernode_public_id = local_args.back();
+    local_args.pop_back();
+
+    bool has_payment_id;
+    crypto::hash8 new_payment_id;
+    if (!cryptonote::get_account_address_from_str_or_url(supernode_public_address, has_payment_id, new_payment_id, m_wallet->testnet(), local_args.back(), oa_prompter))
+    {
+      fail_msg_writer() << tr("failed to parse supernode public wallet address");
+      return true;
+    }
+    
+    if (has_payment_id)
+    {
+      fail_msg_writer() << tr("supernode public wallet address can't be a payment_id") << local_args.back();
+      return true;
+    }
```

Comment:
Can it be a subaddress (coming in the Monero 12/13 merge)?

Reply:
Unfortunately, no. We will check this after merge with Monero. I've created https://github.com/graft-project/GraftNetwork/issues/217 for tracking of this request.